src/keystone.nim

```diff
@@ -142,19 +142,20 @@ template sparc64*(m: type Mode): Mode = (1 shl 3).Mode
 ## SparcV9 mode.
 template v9*(m: type Mode): Mode = (1 shl 4).Mode
 
-proc `or`*(a, b: Mode): Mode {.inline.} =
-  (a.uint or b.uint).Mode
+proc `or`*(a, b: Mode): uint {.inline.} =
```

Comment:
Why use `uint` (or `Mode`) here and in the rest of the document?

Reply:
`or`  operator between Modes causes out of range error, but I dont know why, just a little dirty hack here to make it works

Reply:
Then in that case I think `Mode` should be a `distinct uint`, and its values `littleEndian`, `bigEndian`, and `b{16,32,64}` should be defined like `template xxx*(m: type Mode): Mode = (1 << xxx).Mode`

Reply:
yeah, that's a good idea. Im going to make the changes now